ansible/app_options_files/empty

```diff
@@ -0,0 +1 @@
+
```

Comment:
> I'm considering refacting this completely, because later it turned out that the ethermint app_options part is not compatible with basecoin.

sorry about that. don't think we should put a lot of stuff in here. don't you think it is better to create a separate playbooks/folders for ethermint/basecoin? like `ansible-basecoin`, `ansible-ethermint`?